test/mockRippledTest.ts

```diff
@@ -17,10 +17,14 @@ describe('mock rippled tests', function () {
   })
 
   it('provide bad response shape', async function () {
-    assert.throws(
-      () => this.mockRippled.addResponse('account_info', { data: {} }),
-      Error,
-    )
+    try {
```

Comment:
Why is this preferable to `assert.throws()`

Reply:
The linter complained - `no-unsafe-return`. If you'd rather I disable the error instead of doing this, I can do that.

Reply:
Dont worry about disabling the rule, this is okay.